### PR TITLE
003 - Added create sales employee front end

### PR DESCRIPTION
**Pull request by jemorak**
image

### CODE REVIEW

File: src/controllers/SalesEmployeeController.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import { createSalesEmployee } from "../services/SalesEmployeeService";
+
+export const getSalesEmployeeForm = async (req: express.Request, res: express.Response): Promise<void> => {
+    res.render('salesEmployeeForm.html');
+}
+
+export const postSalesEmployeeForm = async (req: express.Request, res: express.Response): Promise<void> =>{
```
Comment (Leebee1392) [inline src/controllers/SalesEmployeeController.ts:8]: Needs validation

File: src/controllers/SalesEmployeeController.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    try{
+        const id = await createSalesEmployee(req.body,req.session.token);
+        console.log
```
Comment (Leebee1392) [inline src/controllers/SalesEmployeeController.ts:11]: Remove console.log please

File: views/salesEmployeeForm.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (Leebee1392) [inline views/salesEmployeeForm.html:1]: No accessibility

File: views/salesEmployeeForm.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+<html lang="en">
+<head>
+   <title>Document</title>
+</head>
+<body>
+    <h1> Create Product</h1>
```
Comment (Leebee1392) [inline views/salesEmployeeForm.html:7]: Change to Create Sales Employee

**Review by Leebee1392 — APPROVED** [review]

**Review by Adnanhoq — APPROVED** [review]

**Pull request merged**